### PR TITLE
feat(locales): french translation

### PR DESCRIPTION
**Pull request by Crazioz**
hello I send you the french version ^^

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ---------------------------------------------------------------------------------------------
 
 ac = {
-	-- Language for notifications and UI
+	-- Language for notifications and UI (cs/en/et/fr)
```
Comment (antond15) [inline config.lua]: This is unnecessary

File: locales/fr.lua [hunk 2]
```diff
@@ -0,0 +1,31 @@
+return {
+	['ui_join'] = "Joindre à la fréquence	",
```
Comment (antond15) [inline locales/fr.lua]: There are some tabs and spaces that shouldn't be there

**Comment by Crazioz** [comment]
It's good

**Comment by antond15** [comment]
It looks okay 🥖 Thanks

**Pull request merged**